recipes/react-router/README.md

```diff
@@ -0,0 +1,39 @@
+# `react-router` recipe
+
+The `react-router` recipe showcases one of the most powerful ways to implement Puck using to provide an authoring tool for any route in your React Router app.
```

Comment:
Need to do an overhaul if these recipe READMEs at some point, but this is fine for now.